src/libraries/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -694,7 +702,7 @@ internal string GetString()
                     FlushBytes();
                 }
 
-                return _numChars > 0 ? new string(_charBuffer, 0, _numChars) : "";
+                return _numChars > 0 ? _charBuffer.Slice(0, _numChars).ToString() : "";
```

Comment:
```suggestion
                return _charBuffer.Slice(0, _numChars).ToString();
```
No need to special-case empty here. The `ToString` will do it for you